include/maths/CBoostedTreeImpl.h

```diff
@@ -181,9 +184,6 @@ class MATHS_EXPORT CBoostedTreeImpl final {
     //! \return The best hyperparameters for validation error found so far.
     const CBoostedTreeHyperparameters& bestHyperparameters() const;
 
-    //! \return The fraction of data we use for train per fold when tuning hyperparameters.
-    double trainFractionPerFold() const;
```

Comment:
Can you maybe mention in the PR description why you are removing these methods? This doesn't seem to be related to your other changes.

Reply:
I ended up porting #1947 functionality to the feature branch because I need to write out the new section in the meta data for this change as well. In fact, I think it probably makes sense just to implement this on the `feature/incremental-learning` branch because it is primarily needed for this anyway. As part of that change I realised there was no real need to add this to the instrumentation since it is fixed for the duration of train. I'll add above as well.